2-debugging-frames/README.md

```diff
@@ -0,0 +1,41 @@
+# E2: Debugging Frames
+
+Concert designers have dreamed up a revolutionary tippy apple juice concept. The animated mockup is wowing clients across verticals! Most of the ad is built. There are just a couple bugs left to be ironed out. That is your job!
+
```

Comment:
To remove some terms that we probably are more familiar with:

"Our Ad Agency designers have dreamed up a revolutionary new feature for our ads: a tippy apple juice animation! The animated mockup is wowing clients across all of our sites. Most of the code for the ad is written, but there are a couple bugs left to be ironed out. That is your job!"